### DISCUSSION TITLE
[Question] Is there any way to import the quill emoji module in flutter quill?

### DISCUSSION THREAD

**Discussion by gaganyadav80** [singerdmx/flutter-quill#542] [General]
I want to import a quillJS module [quill-emoji](https://github.com/contentco/quill-emoji) in flutter-quill.

Any idea on how to do it or if someone could guide me the steps on how to port this module to flutter-quill just like the quillJS was ported to flutter.

**Comment by singerdmx** [comment]
So you want to implement it?

**Reply by gaganyadav80** [reply]
I want to but I have little to none idea how to do it. If someone can guide me then I can try atleast.
